setup.cfg

```diff
@@ -42,6 +42,9 @@ ignore =
     N812
     # do not use bare 'except'
     E722
+    # Escape characters check.
+    # Conflict with pytest error message regex.
+    W605
```

Comment:
What's this check?

Reply:
It checks if we use escape characters in strings like `\n\r` correctly.
However, in a string which contains a regex may use something like `\.\(\)` which are considered incorrect by W605.